FlowCytometryTools/lib/string.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
```

Comment:
`string` is a built-in python package, so we should avoid shadowing it. try using a different name